### PR TITLE
Mobile 7033 make sure that all protocols are migrated

### PR DESCRIPTION
**Pull request by YuriyVelichkoOpenX**
Part of protocols are  migrated

### CODE REVIEW

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/AdLoading/PBMInterstitialAdLoader.m [hunk 1]
```diff
@@ -71,17 +69,17 @@ - (void)reportSuccessWithAdObject:(id)adObject adSize:(nullable NSValue *)adSize
         }];
         return;
     }
-    if ([adObject conformsToProtocol:@protocol(PBMInterstitialEventHandler)]) {
-        id<PBMInterstitialEventHandler> const eventHandler = (id<PBMInterstitialEventHandler>)adObject;
+    if ([adObject conformsToProtocol:@protocol(InterstitialEventHandlerProtocol)]) {
+        id<PBMInterstitialAd> const eventHandler = (id<PBMInterstitialAd>)adObject;
```
Comment (VadimKhohlovOpenX) [inline PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/AdLoading/PBMInterstitialAdLoader.m:73]: Here we are checking confirmation to InterstitialEventHandlerProtocol but casting to PBMInterstitialAd. It's a little bit confusing. Let's cast to InterstitialEventHandlerProtocol.

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/AdLoading/PBMInterstitialAdLoader.m [hunk 2]
```diff
@@ -71,17 +69,17 @@ - (void)reportSuccessWithAdObject:(id)adObject adSize:(nullable NSValue *)adSize
...
         [self.delegate interstitialAdLoader:self
                                    loadedAd:^(UIViewController *targetController) {
             [eventHandler showFromViewController:targetController];
         } isReadyBlock:^BOOL{
             return eventHandler.isReady;
         }];
         return;
-    } else if ([adObject conformsToProtocol:@protocol(PBMRewardedEventHandler)]) {
-        id<PBMRewardedEventHandler> const eventHandler = (id<PBMRewardedEventHandler>)adObject;
+    } else if ([adObject conformsToProtocol:@protocol(RewardedEventHandlerProtocol)]) {
```
Comment (VadimKhohlovOpenX) [inline PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/AdLoading/PBMInterstitialAdLoader.m:81]: The same as for the InterstitialEventHandlerProtocol & PBMInterstitialAd above.

**Review by VadimKhohlovOpenX — COMMENTED** [review]
See my comments

**Comment by YuriyVelichkoOpenX** [comment]
All changes and fixes according to comments are introduced in this PR
https://github.com/openx/prebid-mobile-ios/pull/27

**Pull request closed without merging**